src/components/vm/disks/vmDisksCard.tsx

```diff
@@ -171,45 +169,25 @@ export const VmDisksCard = ({
     vm,
     vms,
     disks,
-    renderCapacity,
     supportedDiskBusTypes,
     storagePools,
     onAddErrorNotification
 } : {
     vm: VM,
     vms: VM[],
     disks: VMDiskWithData[],
-    renderCapacity: boolean,
     supportedDiskBusTypes: string[],
     storagePools: StoragePool[],
     onAddErrorNotification: (notification: Notification) => void,
 }) => {
     const [openActions, setOpenActions] = useState(new Set());
-    let renderCapacityUsed = false;
-    let renderAccess = false;
-    let renderAdditional = false;
     const columnTitles: (string | ListingTableColumnProps)[] = [_("Device")];
     const idPrefix = `${vmId(vm.name)}-disks`;
 
     if (disks && disks.length > 0) {
-        renderCapacityUsed = disks.some(disk => (!!disk.used));
-        renderAccess = disks.some(disk => (typeof disk.readonly !== "undefined") || (typeof disk.shareable !== "undefined"));
-        renderAdditional = disks.some(disk => (disk.driver.cache || disk.driver.io || disk.driver.discard || disk.driver.errorPolicy || disk.driver.type || disk.serial));
-
-        if (renderCapacity) {
-            if (renderCapacityUsed) {
-                columnTitles.push(_("Used"));
```

Comment:
I feel a bit sad about that. The purpose of Cockpit is troubleshooting and seeing the current state. Seeing that a disk now is 90% full is much more interesting than the static absolute size. It could show a percentage here if you don't want to spend the extra column, or used and total in the same column, like "1.5 GiB/2 GiB"?

Reply:
But that doesn't mean it shouldn't be in the table columns, hmm.... let me rearrange things a bit, I think I have a plan...

Reply:
Ah, so that "Used" column was the allocation on the host, i.e. not really relevant for the guest? Then I agree, that doesn't need to go into the table, and in the details the label should be improved to point that out (or perhaps a popover text?) I figure detecting what "used" is in the guest requires cooperation with the guest OS and is difficult to figure out.

Reply:
> Ah, so that "Used" column was the allocation on the host, i.e. not really relevant for the guest?

Correct.